02-oop-inheritance/Module_pattern/script.js

```diff
@@ -0,0 +1,57 @@
+
+var inheritPrototype = function (childClass, parentClass){
+  var copyOfParent = Object.create(parentClass.prototype);
+  copyOfParent.constructor = childClass;
+  childClass.prototype = copyOfParent;
+};
+
+var social = function () {
```

Comment:
Always capitalize classes. ``social`` should be ``Social``.
Also, it's convenient to have ``share`` and ``like`` methods defined within ``Social``'s prototype. We gave a brief insight about this last time we met, remember that if you define methods inside your class, each time you instantiate it, you'll be creating those functions.